src/telegram_py_bot.py

```diff
@@ -1,6 +1,24 @@
 from telegram.ext import ApplicationBuilder, CommandHandler, MessageHandler, filters
 from qr_code_generator import QR
```

Comment:
Puoi rimuovere la riga 2 `from qr_code_generator import QR`, dato che la funzione QR si trova già dentro il file del bot. In questo modo pylint non dovrebbe più dare problemi